lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -445,7 +445,8 @@ protected function transformSignup($data, $current) {
    */
   protected function transformUser($data) {
     return [
-      'id' => $data->id,
+      'id' => $data->drupal_id,
```

Comment:
Its not my favorite naming, but I'd probably vote for using `drupal_id` for the Drupal ID just for consistency with how we refer to it elsewhere. And then just make sure to document that it's not a preferred way to identify a user & may be removed in the future.